docs/SCTValidation.md

```diff
@@ -196,24 +196,27 @@ checks need to be done as if the pre-issuer were not present.  For example:
    theoretically not allow the pre-issuer to be a CA certificate).
 
 So to sum up *pre-issued* precertificates:
- - The CA builds a precertificate version of the certificate that includes the
-   poison.
- - Next, the CA modifies the Issuer and (optional) Authority Key Identifier
-   extension in the precertificate to match the pre-issuer rather than the true
-   issuer.
- - The CA signs the resulting precertificate using the pre-issuer's key.
- - The submits the whole chain (precert, pre-issuer, true-issuer, ... root) to
-   the Log.
- - The Log removes the poison and extracts the inner `tbsCertificate`.
- - The Log notices that the direct issuer has the Certificate Transparency
-   extended key usage, so treats the next entry in the chain as the true issuer.
- - The Log updates the Issuer and (optional) Authority Key Identifier extension
-   to match the true issuer rather than the pre-issuer.
- - The Log combines the resulting inner `tbsCertificate` with the hash of the
-   (true) issuer's key, to form the core `PreCert` data that the Log deals with.
- - The Log sends an SCT back to the CA, which includes a timestamp as usual but
-   has a signature over data including the `PreCert` bundle of `tbsCertificate`
-   and issuer key hash.
+ - The CA:
```

Comment:
would it be clearer if these were on a numbered list to suggest ordering?

Reply:
I tried it locally but it looked really odd with the incomplete sentence style of bullets.